alerter.js

```diff
@@ -20,7 +26,8 @@ function alertInCelcius(farenheit) {
     }
 }
 
-alertInCelcius(400.5);
+alertInCelcius(400.5, networkAlertStub);
+expect(alertFailureCount).equals(1);

```

Comment:
This `expect` checks if the previous one alerted. However, there is no check to see that the next one doesn't alert. The test will pass even if it alerts all the time.